docs/sources/next/javascript-api/k6-experimental/browser/locator/getattribute.md

```diff
@@ -19,9 +19,9 @@ Returns the element attribute value for the given attribute name.
 
 ### Returns
 
-| Type   | Description                         |
-| ------ | ----------------------------------- |
-| string | The value of the attribute or null. |
+| Type              | Description                                                                                               |
+| ----------------- | --------------------------------------------------------------------------------------------------------- |
+| `Promise<string>` | A Promise that fulfills with the string value of the attribute or `null` if the attribute is not present. |
```

Comment:
```suggestion
| `Promise<string | null>` | A Promise that fulfills with the string value of the attribute or `null` if the attribute is not present. |
```

WDYT?

Reply:
Shouldn't this "ideally" always return a `string`? Maybe we have a bug in the code since we return an [`any`](https://github.com/grafana/xk6-browser/blob/99686dfc84e86e0d02be70323a5cd4902c306438/common/locator.go#L325).

Reply:
I think it might be useful to distinguish between the attribute being present and it having the value `""`. I might want to check that the correct `aria` attribute was added to the element after some action.

Reply:
Makes sense 👍 Let's make this change, then.